src/o365/spo/commands/list/list-sitescript-get.ts

```diff
@@ -0,0 +1,240 @@
+import auth from '../../SpoAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import * as request from 'request-promise-native';
+import {
+  CommandOption,
+  CommandValidate,
+  CommandError
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import Utils from '../../../../Utils';
+import { Auth } from '../../../../Auth';
+import { ContextInfo, 
+  // ClientSvcResponse, 
+  // ClientSvcResponseContents 
+} from '../../spo';
+import { ListInstance } from './ListInstance';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+}
+
+class SpoListSitescriptGetCommand extends SpoCommand {
```

Comment:
Site script are two words so we should spell the command class name like `SpoListSiteScriptGetCommand`